th2_data_services_lwdp/commands/http.py

```diff
@@ -262,25 +267,26 @@ def __init__(
 
         Args:
             book_id (str): Book ID.
-            start_timestamp (datetime): Start Timestamp.
-            end_timestamp (datetime): End Timestamp.
+            start_timestamp (datetime): Start Timestamp. API expects timestamp in milliseconds.
+            end_timestamp (datetime): End Timestamp. API expects timestamp in milliseconds.
             sse_handler (Optional, IStreamAdapter): SSE Events Handler. Defaults To `SSEAdapter`.
             cache (Optional, bool): Cache Status. Defaults To `False`.
             buffer_limit: SSEAdapter BufferedJSONProcessor buffer limit.
         """
+        _check_milliseconds(start_timestamp)
+        _check_milliseconds(end_timestamp)
         self._sse_handler = sse_handler or get_default_sse_adapter(buffer_limit=buffer_limit)
         super().__init__(self._sse_handler, cache)
         self._book_id = book_id
-        self._start_timestamp = start_timestamp
-        self._end_timestamp = end_timestamp
+        self._start_timestamp = DatetimeConverter.to_milliseconds(start_timestamp)
+        self._end_timestamp = DatetimeConverter.to_milliseconds(end_timestamp)
 
     def _get_urls(self, api: HTTPAPI):
-        # Single URL, But Parent Class Attribute Needs List
         return [
             api.get_url_get_pages_info(
                 self._book_id,
-                _datetime2ms(self._start_timestamp),
-                _datetime2ms(self._end_timestamp),
+                DatetimeConverter.to_milliseconds(self._start_timestamp),
+                DatetimeConverter.to_milliseconds(self._end_timestamp),
```

Comment:
Why do you convert it twice?
Why you don't test your code before sending to review?

Reply:
Sorry, I think during merge I converted time in different places and it must have slipped.
I ran the code but it didnt catch it as error because of way utils.DatetimeConverter handled timestamps.